reposerver/src/main/scala/com/advancedtelematic/tuf/reposerver/http/RoleChecksumHeader.scala

```diff
@@ -0,0 +1,30 @@
+package com.advancedtelematic.tuf.reposerver.http
+
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.server.{Directive0, Directive1, Directives}
+import com.advancedtelematic.libats.data.DataType.ValidChecksum
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.refineV
+import cats.syntax.either._
+
+object RoleChecksumHeader {
+  import Directives._
+
+  type RoleChecksum = Refined[String, ValidChecksum]
+
+  private val HEADER_NAME: String = "x-ats-role-checksum"
```

Comment:
Could be public and used where the string is used currently.

Reply:
No because this is in the reposerver package and it cannot be elsewhere because it uses akka which is not a dependency for all submodules